MaterialLib/Fluid/Density/IdealGasLaw.h

```diff
@@ -0,0 +1,99 @@
+/*!
+   \file  IdealGasLow.h
+   \brief Declaration of class LIdealGasLow for fluid density by the ideal gas
+   law
+          depending on one variable linearly.
+
+   \author Wenqing Wang
+   \date Jan 2015
+
+   \copyright
+    Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+               Distributed under a Modified BSD License.
+               See accompanying file LICENSE.txt or
+               http://www.opengeosys.org/project/license
+ */
+#ifndef IDEAL_GAS_LAW_H_
```

Comment:
It is generated by a GUI automatically. I think  this header guard name is specific, it would not conflict with those in other 3rd party codes. 

I suggest to use OGS_as the prefix of the  the header guards, and it could be done in a PR.